synapse/storage/databases/main/media_repository.py

```diff
@@ -520,11 +533,15 @@ async def get_cached_remote_media(
                 "upload_name",
                 "created_ts",
                 "filesystem_id",
+                "last_access_ts",
                 "quarantined_by",
             ),
             allow_none=True,
             desc="get_cached_remote_media",
         )
+        if row is None:
+            return row
+        return RemoteMedia(media_origin=origin, media_id=media_id, **row)
```

Comment:
I guess this is going to clash with #16612?

Reply:
#16612 is based on top of this branch. 😄